.github/workflows/.github/workflows/linters.yml

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Please your linter.yml file is created but it is empty. Kindly make sure that you add the right setup. You can check and get the right html and css linter files from [here](https://github.com/microverseinc/linters-config)

Reply:
Thank you so much for your code review. I have followed the steps and pasted all the config files to the linter file. Then, I added the folder of .github/workflows/linters.yml to .gitignore file.